packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -78,29 +66,14 @@ export interface VideoGalleryProps {
   remoteParticipants?: VideoGalleryRemoteParticipant[];
   /** List of dominant speaker userIds in the order of their dominance. 0th index is the most dominant. */
   dominantSpeakers?: string[];
-  /** Local video view options */
-  localVideoViewOption?: VideoStreamOptions;
-  /** Remote videos view options */
-  remoteVideoViewOption?: VideoStreamOptions;
-  /** Callback to create the local video stream view */
-  onCreateLocalStreamView?: (options?: VideoStreamOptions) => Promise<void>;
-  /** Callback to dispose of the local video stream view */
-  onDisposeLocalStreamView?: () => void;
-  /** Callback to render the local video tile*/
-  onRenderLocalVideoTile?: (localParticipant: VideoGalleryLocalParticipant) => JSX.Element;
-  /** Callback to create a remote video stream view */
-  onCreateRemoteStreamView?: (userId: string, options?: VideoStreamOptions) => Promise<void>;
-  /** Callback to render a remote video tile */
-  onRenderRemoteVideoTile?: (remoteParticipant: VideoGalleryRemoteParticipant) => JSX.Element;
-  onDisposeRemoteStreamView?: (userId: string) => Promise<void>;
-  /** Callback to render a particpant avatar */
-  onRenderAvatar?: OnRenderAvatarCallback;
-
-  /**
-   * Whether to display a mute icon beside the user's display name.
-   * @defaultValue `true`
-   */
-  showMuteIndicator?: boolean;
+  /** Callback to render participants in VideoGallery */
+  onRenderTile: (
+    props: VideoGalleryLocalParticipant | VideoGalleryRemoteParticipant,
+    type: 'participant' | 'screenshare' | 'localParticipant' | 'localScreenshare'
```

Comment:
just curious if we should have remote/local or just, as you did, no-prefix/local

Reply:
Looking into this...